.clang-format

```diff
@@ -0,0 +1,111 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
+AccessModifierOffset: -1
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlines: Right
+AlignOperands:   true
```

Comment:
Having some values indented a bit while all others just use a single whitespace does is not well readable for me. I would either indent all values to be on the same line or not indent them at all.

Reply:
The indentation is produced by clang-format's dump option. We could make it more readable, but note that if we dump the Google styles (they changes between different versions of clang-format) the next time we will have to change those again.